data/en/footer.yml

```diff
@@ -42,4 +42,24 @@ footer:
     - name: Open Source
       link: /opensource
     - name: Resources
-      link: /resources
\ No newline at end of file
+      link: /resources
+
+#Social Media Links
+  social:
```

Comment:
Adding social media links to the data file should make it easier for anyone to make updates.